examples/jsm/interactive/HTMLMesh.js

```diff
@@ -445,6 +445,17 @@ function html2canvas( element ) {
 
 			}
 
+			if ( element instanceof HTMLImageElement ) {
+
+				element.addEventListener( 'load', () => {
```

Comment:
Since you are using the `load` event listener: What would happen if the image is already loaded?

Reply:
Ah shoot you're right, I totally forgot about that. It's late right now (probably why I didnt think of it) but I'll fix this in the morning.